lib/record_list.rb

```diff
@@ -0,0 +1,32 @@
+require_relative 'entry'
+
+class RecordList
```

Comment:
I see that you removed `Account`, which is great.  Is there value in a class that's kind of a UI class that lets the user do everything on one object, and that that UI object knows how to combine RecordList, statement, Entry etc in clear ways that use the domain language?